index.html

```diff
@@ -3225,36 +3229,38 @@ <h3>Terms of Use</h3>
     "https://www.w3.org/ns/credentials/v2",
     "https://www.w3.org/ns/credentials/examples/v2"
   ],
-  "id": "http://university.example/credentials/3732",
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
-  "issuer": "https://university.example/issuers/14",
-  "validFrom": "2010-01-01T19:23:24Z",
+  "id": "urn:did:123456",
+  "type": [
+    "VerifiableCredential",
+    "EbsiToUExample"
```

Comment:
```suggestion
    "EbsiTermsOfUseExample"
```

Avoid too many acronyms in a row.

Reply:
done